scripts/cross-rv32imac_zicsr_zbb.txt

```diff
@@ -14,8 +14,8 @@ cpu = 'riscv32'
 endian = 'little'
 
 [properties]
-c_args = ['-msave-restore', '-fshort-enums', '-march=rv32imac_zicsr', '-mabi=ilp32']
-c_link_args = ['-msave-restore', '-fshort-enums', '-march=rv32imac', '-mabi=ilp32']
+c_args = ['-msave-restore', '-fshort-enums', '-march=rv32imac_zicsr_zbb', '-mabi=ilp32']
+c_link_args = ['-msave-restore', '-fshort-enums', '-march=rv32imac_zbb', '-mabi=ilp32']
```

Comment:
Hrm. Does this still work if c_args and c_link_args both use `-march=rv32imac_zicsr_zbb`? Would be nice to fix this inconsistency.

Reply:
Now, the inconsistency between c_args and c_link_args is fixed